### DISCUSSION TITLE
Actividad 0 - Abregu Candela

### DISCUSSION THREAD

**Discussion by CandelAbregu** [frlp-utn-ingsoft/catedra#82] [Show and tell]
https://github.com/CandelAbregu/Actividad-0.git

**Comment by ivanduva** [comment]
Buenas,
Vemos dos problemas en la actividad:

- El principal, no cumple con la consigna porque hay un solo commit. Además el commit no es atómico y el mensaje no es descriptivo de la funcionalidad.
- La app no funciona.

Te vamos a pedir que lo empieces de vuelta en otro repositorio, cumpliendo la consigna y probando la app a medida que vas avanzando.
Si tenés dudas las vamos resolviendo.
Saludos
